deploy/cloudformation/parameters.json

```diff
@@ -0,0 +1,62 @@
+[
+    {
+        "ParameterKey": "KeyName",
+        "ParameterValue": "kirill"
```

Comment:
Hi Kiriil! ;)

(Can we change to something less... personal)?

Reply:
Sure.